frontend/src/component/changeRequest/ChangeRequestOverview/ReviewButton/ReviewButton.tsx

```diff
@@ -117,6 +137,16 @@ export const ReviewButton: FC<{ disabled: boolean }> = ({ disabled }) => {
                                     <MenuItem onClick={onApprove}>
                                         Approve changes
                                     </MenuItem>
+                                    <ConditionallyRender
+                                        condition={Boolean(
+                                            uiConfig?.flags?.changeRequestReject
+                                        )}
+                                        show={
```

Comment:
this will open a modal in the next PR. For now it makes a direct change to test it E2E